### ISSUE TITLE
built-in support/examples for scraping ASP.net pages

### ISSUE THREAD

**Issue by jamesturk** [jamesturk/spatula#24]
A common challenge people run into is scraping pages built with ASP.net (or similar technology) where there is a ton of hstate kept in forms and cookies.  I've had some success doing this with a custom source (https://github.com/openstates/openstates-scrapers/blob/main/scrapers_next/hi/people.py#L6) but we can formalize or at least document the approach.

If you're seeing this and would find it useful, it'd be great to get an example of a page that would need this technique as well as a 👍 or any other comments you have.

**Comment by mscarey** [comment]
Yes, I was wondering if this kind of approach could be used for search forms like the ones Tyler Tech uses. For instance, https://odysseypa.traviscountytx.gov/JPPublicAccess/Search.aspx?ID=900.
